scripts/dev/estimate-usage/README.md

```diff
@@ -0,0 +1,103 @@
+# `estimate-usage` scripts
+These scripts exist to allow developers to determine approximately how much disk
```

Comment:
Minor: Update commit header to `Add README for the estimate-usage scripts`. Try to keep the commit header length under 50 for all commits.

Reply:
Shortened other commit messages too.